ui/src/main/js/page/channel/jira/server/JiraServerDistributionConfiguration.js

```diff
@@ -6,16 +6,50 @@ import CheckboxInput from 'common/component/input/CheckboxInput';
 import TextInput from 'common/component/input/TextInput';
 import CollapsiblePane from 'common/component/CollapsiblePane';
 import FieldMappingField from 'common/component/input/FieldMappingField';
+import { DISTRIBUTION_COMMON_FIELD_KEYS } from 'page/distribution/DistributionModel';
+import EndpointSelectField from '../../../../common/component/input/EndpointSelectField';
+import { createReadRequest } from '../../../../common/util/configurationRequestBuilder';
 
 const JiraServerDistributionConfiguration = ({
-    data, setData, errors, readonly
+    csrfToken, data, setData, errors, readonly
 }) => {
+    const readRequest = () => {
+        const apiUrl = '/alert/api/configuration/jira_server?pageNumber=0&pageSize=25';
```

Comment:
I know this is probably very edge case, but would we need to state anywhere that we have an upper limit of 25 Jira Servers?

Reply:
I added to the description we limit to the first 25 Jira servers